codewof/templates/users/dashboard.html

```diff
@@ -30,11 +30,24 @@ <h2>Your questions for today</h2>
             <strong>Sorry!</strong> We currently don't have questions for you to do today.
           </div>
         {% endif %}
+        <p>You've solved {{number_days_used}} questions in the last month!</p>
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/65398369-da004200-de0a-11e9-8789-b58b9565533d.png)
@courtneycb how do you deal with the non-plural version of sentences again?

Reply:
Also, shouldn't `number_days_used` be a variable related to the number of questions answered?

Reply:
You should follow this doucmentation for pluralisation:
https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#pluralize